### ISSUE TITLE
Area Types: Four new tree types for use in CMIP7

### ISSUE THREAD

**Issue by martinjuckes** [cf-convention/vocabularies#227]
**Proposee**: Martin Juckes, Daniele Peano

**Date** : 2024-09024

For each term please try to give the following:

**- Terms** 

- broadleaf_deciduous_trees
- broadleaf_evergreen_trees
- needleleaf_deciduous_trees
- needleleaf_evergreen_trees

**- Description** 

The distinction broadleaf vs needleleaf and deciduous vs. evergreen have important implications for the carbon cycle. The four categories have been used in the CMIP6 without the blessing of CF. They are likely to be used more extensively in CMIP7, so it would be very helpful to have them added to the Area Types table.

**Comment by JonathanGregory** [comment]
I agree that these four tree types are sensible additions. I notice that the existing deciduous/evergreen tree area types are also distinguished as primary/secondary. Do CMIP6 and CMIP7 use that distinction?

**Comment by martinjuckes** [comment]
Those four area types (combinations of deciduous/evergreen and primary/secondary) were used in CMIP6, but appear to have had significantly lower usage than the broad-/narrow-leaf and evergreen/deciduous combinations. The  deciduous/evergreen and primary/secondary variants have not yet been requested for CMIP7.

**Comment by daniele-peano** [comment]
The original request of 
Broadleaf Deciduous
Broadleaf Evergreen
Needleleaf Deciduous
Needleleaf Evergreen
derives from a proposed opportunity to study plant phenology. 
In that case, the distinction between deciduous and evergreen plants is more relevant than the primary/secondary one.

**Comment by JonathanGregory** [comment]
@martinjuckes @daniele-peano Thanks for explaining. That satisfies my curiosity!

**Comment by japamment** [comment]
Dear Martin and Daniele, @martinjuckes @daniele-peano 

Thank you for these proposals for four new area types. I agree that they would make sensible additions to the area type table.

We should add descriptions for the terms. We don't currently have any descriptions for broadleaf and needleleaf, nor indeed evergreen, so I have added some text based on online references.

(1) `broadleaf_deciduous_trees`
**Description:** Broadleaf trees are characterized by leaves that are broad, flat and thin with networked veins (reference: https://treeterms.co.uk/broadleaf/). Deciduous trees lose their leaves seasonally, for example, during winter in high latitudes or following seasonal variations in rainfall.

(2) `broadleaf_evergreen_trees`
**Description:** Broadleaf trees are characterized by leaves that are broad, flat and thin with networked veins (reference: https://treeterms.co.uk/broadleaf/). Evergreen plants retain their leaves through the year and into the following growing season (reference: https://www.britannica.com/plant/evergreen-plant).

(3) `needleleaf_deciduous_trees`
**Description:** Needleleaf trees, often referred to as conifers, have long, thin needle-shaped leaves that are well adapted to retaining water. Deciduous trees lose their leaves seasonally, for example, during winter in high latitudes or following seasonal variations in rainfall.

(4) `needleleaf_evergreen_trees`
**Description:** Needleleaf trees, often referred to as conifers, have long, thin needle-shaped leaves that are well adapted to retaining water. Evergreen plants retain their leaves through the year and into the following growing season (reference: https://www.britannica.com/plant/evergreen-plant).

Are you happy with these descriptions? If so, I think these terms can be accepted for inclusion in the area type table.

Best wishes,
Alison

**Comment by daniele-peano** [comment]
Dear Alison @japamment,
These descriptions illustrate the main characteristics and differences between these four plant types.
@martinjuckes Martin any specific information is usually added in the area type descriptions that is missing here?

Thank you very much!
Daniele

**Comment by japamment** [comment]
Dear Daniele, @daniele-peano 

Thank you for confirming - I think we have all the necessary information now.

If no further comments are received in the next seven days, these proposals will be accepted for publication in the CF area type table.

Best wishes,
Alison

**Comment by japamment** [comment]
The terms in this issue have all been published in v12 of the area type table so closing this one now.
